test/integration/krane_deploy_test.rb

```diff
@@ -199,6 +199,24 @@ def test_refuses_deploy_to_protected_namespace_with_override_if_pruning_enabled
     @namespace = generated_ns
   end
 
+  def test_deploying_to_protected_namespace_does_not_prune
```

Comment:
The `krane` version of this didn't need to be serial, because it didn't need to stub protected namespaces list anymore (it used to be a constant)